src/integer/z/properties.rs

```diff
@@ -118,3 +132,26 @@ mod test_distance {
         assert_eq!(Z::ZERO, a.distance(&a));
     }
 }
+
+#[cfg(test)]
+mod test_is_prime {
+    use super::Z;
+
```

Comment:
test for non-prime is missing

Reply:
There is a test for that in the module under the function name "non_prime_detection"